### DISCUSSION TITLE
Support for CloudWatch Dynamic Labels

### DISCUSSION THREAD

**Discussion by dconrique01** [grafana/grafana#38337] [Data Sources]
**What would you like to be added**:
The customer would like to have support for dynamic labels for the CloudWatch data source in Grafana. 

**Why is this needed**:
Per the customer, CloudWatch doesn't return proper dimension value if namespace contains only one instance. AWS recommends to use dynamic labels per: https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/graph-dynamic-labels.html#dynamic-label-syntax. These label options are currently not supported in Grafana.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by fridgepoet** [comment]
Thanks! We aim to get this into Grafana 9.0 https://github.com/grafana/grafana/issues/48434

**Comment by sarahzinger** [comment]
Hey all this has been released! Closing this discussion!
